src/utils/constants.ts

```diff
@@ -305,3 +305,6 @@ export const CONTEXT = {
   view: "view",
   edit: "edit"
 };
+export const CONCEPT_GENERAL: string[] = [
```

Comment:
I'd almost think this was better suited for `src/app/components/constants.ts`.

Reply:
Thanks, @ibacher just that I saw as if the backend expects number 1 or 0 🤔

Reply:
@hadijahkyampeire That would explain why changing `true` to `false` didn't do anything. I would think the best way to handle that would be to map from a boolean to an integer in `api.ts` (so when we make the call to the backend).